app/styles/_base.scss

```diff
@@ -46,6 +46,10 @@ body > div > div > div { // FIXME: add .full-height-container to each page root
 
   @media #{$mq-laptop} {
     padding: 0 $wrap-desktop;
+
+    &.-big {
```

Comment:
As we talked previously, it would be better set a container with a `max-width` instead of modifying wrap (I'm not saying to not modify them, but if it's not necessary better leave them).